applications/dairdrop.md

```diff
@@ -0,0 +1,232 @@
+
+Below we provide an **example roadmap**. In the descriptions, it should be clear how your project is related to Substrate, Kusama or Polkadot. We *recommend* that teams structure their roadmap as 1 milestone ≈ 1 month. -->
+
+<!-- > :exclamation: If any of your deliverables are based on somebody else's work, make sure you work and publish *under the terms of the license* of the respective project and that you **highlight this fact in your milestone documentation** and in the source code if applicable! **Projects that submit other people's work without proper attribution will be immediately terminated.** -->
+
+### Overview
+
+- **Total Estimated Duration:** 3 months
+- **Full-Time Equivalent (FTE):**  3,5
+<!-- - Average number of full-time employees working on the project throughout its duration (see [Wikipedia](https://en.wikipedia.org/wiki/Full-time_equivalent), e.g. 2 FTE) -->
+- **Total Costs:** 20,000
+- **DOT %:** 50%
+
+### Milestone 1 — On-chain pallets and Ink contracts, airdrop indexer, API services, and more.
+
+- **Estimated duration:** 1.5 month
+- **FTE:**  2,0
+- **Costs:** 10,000 USD
+
+> :exclamation: **The default deliverables 0a-0d below are mandatory for all milestones**, and deliverable 0e at least for the last one.
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0  |
+| **0b.** | Documentation | Create detailed documentation for each module. |
+| **0c.** | Testing and Testing Guide | Over 70% code test coverage. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 1. | Substrate module: airdorp | An airdrop module that provides all on-chain functionalities related to airdrops |
+| 2. | Smart contracts:  airdorp-ink| Implementation of the airdrop module using Ink. |
+| 3. | indexer| An on-chain airdrop activity indexer that monitors on-chain airdrop status in real time.|
+| 3. | backend | Provide all APIs needed for the app, including calculating the MD5 hash of the airdrop file, the Merkle root of the airdrop, Merkle proofs, and airdrop status. |
+| 4. | Substrate chain | Dota 2.0 test chain used for testing airdrop functionality and preparing for the launch of the Dota 2.0 mainnet. |
+
+
+### Milestone 2 — UI/UX design and front-end development.
+
+- **Estimated Duration:** 1.5 month
+- **FTE:**  1,5
+- **Costs:** 10,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0  |
+| **0b.** | Documentation | Provide documentation for the app. |
+| **0c.** | Testing and Testing Guide | Provide manual test cases as much as possible. |
```

Comment:
I'd suggest to reword this. "as much as possible" is a very subjective term.

Reply:
> I'd suggest to reword this. "as much as possible" is a very subjective term.

I agree. I will  update soon.